cpp/examples/arrow/row_wise_conversion_example.cc

```diff
@@ -176,15 +188,49 @@ arrow::Status ColumnarTableToVector(const std::shared_ptr<arrow::Table>& table,
 
 int main(int argc, char** argv) {
   std::vector<data_row> rows = {
-      {1, 1.0, {1.0}}, {2, 2.0, {1.0, 2.0}}, {3, 3.0, {1.0, 2.0, 3.0}}};
-
+      {1, 1, {10.0}}, {2, 3, {11.0, 12.0, 13.0}}, {3, 2, {15.0, 25.0}}};
   std::shared_ptr<arrow::Table> table;
-  EXIT_ON_FAILURE(VectorToColumnarTable(rows, &table));
-
   std::vector<data_row> expected_rows;
-  EXIT_ON_FAILURE(ColumnarTableToVector(table, &expected_rows));
 
-  assert(rows.size() == expected_rows.size());
+  arrow::Result<std::shared_ptr<arrow::Table>> table_result = 
+          VectorToColumnarTable(rows);
+  if(table_result.ok()){
+          table = std::move(table_result).ValueOrDie();
```

Comment:
Can you try to follow the [coding conventions](https://arrow.apache.org/docs/developers/cpp/development.html#code-style-linting-and-ci)? Specifically:
* use 2-space indentation
* use whitespace adequately around keyword and operators, for example `if (table_result.ok()) {` and not `if(table_result.ok()){`

Reply:
ran linter on the code